finetune/util/table_labeler.py

```diff
@@ -389,22 +388,26 @@ def predict_from_file(
         scheduler: Scheduler,
         return_negative_confidence=False,
         config_overrides=None,
+        cache_key=None,
     ):
-        etl = get_etl_from_file(model_file_path)
+        etl = scheduler.etl_cache(cache_key)
+        # etl = get_etl_from_file(model_file_path)
```

Comment:
can you remove this commented line?

Reply:
should i remove the get_etl_from_file as well since it is no longer being used?

Reply:
left it in for now in case it ever needs to be used again